nymms/schemas/__init__.py

```diff
@@ -105,12 +117,43 @@ def increment_attempt(self):
 
 class Result(StateModel, OriginModel):
     id = StringType(required=True)
-    timestamp = TimestampType(default=time.time)
+    timestamp = TimestampType(default=arrow.get)
     output = StringType()
     task_context = JSONType()
 
```

Comment:
Wow, really?  That means task_context's are getting over 1k in size, that's huge.  Honestly I've been wondering if I need to start to define the context a bit more, there are certain parts of it that are probably used often enough that they could become a standard part of the Result.

Thanks - let me update the code with this.